src/CCDReduction.jl

```diff
@@ -18,7 +19,10 @@ export subtract_bias,
        combine,
        subtract_dark,
        subtract_dark!,
-       fitscollection
+       fitscollection,
+       array,
+       filename,
+       images
```

Comment:
we should use plural for all of these functions- `images` is an iterator of images, so we use plural. `array` is an iterator of arrays, yet it's being labeled `array`.

On top of that, `array` and `filename` as function names is just asking for confusion/name conflicts. `arrays` and `filenames` is a little less likely.